#! /bin/sh /usr/share/dpatch/dpatch-run
## 99_epg-fix.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes small bug in event handling - probably fixed in next release

@DPATCH@
diff -urNad vdr-1.3.41~/epg.c vdr-1.3.41/epg.c
--- vdr-1.3.41~/epg.c	2006-01-29 15:17:33.000000000 +0100
+++ vdr-1.3.41/epg.c	2006-02-01 01:41:17.000000000 +0100
@@ -321,7 +321,7 @@
                              }
                           }
                        break;
-             case 'e': if (!Event->Title())
+             case 'e': if (Event && !Event->Title())
                           Event->SetTitle(tr("No title"));
                        Event = NULL;
                        break;
